.github/workflows/deploy.yml

```diff
@@ -163,3 +163,15 @@ jobs:
           SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
           SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
           SENTRY_PROJECT: ${{ secrets.SENTRY_PROJECT }}
+  e2e:
+    runs-on: ubuntu-20.04
+    name: E2E on Chrome
+    needs: deploy
+    steps:
+      - uses: actions/checkout@v2
+      - uses: cypress-io/github-action@v2
+        with:
+          spec: cypress/integration/smoke/*.spec.js
+          browser: chrome
+          record: false
+          config: baseUrl=https://pr${{ github.event.number }}--safereact.review-safe.gnosisdev.com/app/
```

Comment:
If it works again, let's keep it! :)

Reply:
ok!

Reply:
Btw really nice that you made it a separate workflow 👍